#Scenes/SceneScripts/TestingScene.gd

```diff
@@ -0,0 +1,7 @@
+extends Node2D
+
+## * We do this to properly init the battle state
+## Before, the testing scene was the main scene so it was properly started by the phase manager
+## Now that the main scene is the main menu and we load the testing scene from there, we init to properly init the game state 
+func _ready() -> void:
+	PhaseManager.initialize_game()
```

Comment:
Since we have this now, should we remove the `initialize_game` call in PhaseManager _ready?

Reply:
I think we could yes... since it will be called upon scene loading
It just won't work if you directly load the scene from the start of the game instead of loading it from the global loader